packages/blocks/src/store/private-selectors.js

```diff
@@ -160,3 +160,27 @@ export const getSupportedStyles = createSelector(
 	},
 	( state, name ) => [ state.blockTypes[ name ] ]
 );
+
+/**
+ * Returns the bootstrapped (initialized statically from server) block type for a give block name.
```

Comment:
Noting, that we also would use it previously for the metadata loaded from `block.json` after a light sanitization and applying translation helpers to some fields.